python/ql/lib/semmle/python/dataflow/new/internal/TypeTracker.qll

```diff
@@ -545,6 +575,13 @@ module TypeTracker {
    * Gets a valid end point of type tracking.
    */
   TypeTracker end() { result.end() }
+
+  /**
+   * INTERNAL USE ONLY.
+   *
+   * Gets a valid end point of type tracking with the call bit set to the given value.
+   */
+  predicate end = Cached::noContentTypeTracker/1;
```

Comment:
This bit doesn't seem to be used yet, can we remove it until then?

I guess that could be said about _all_ the changes to this Python file, this one just stood out as something that needed a motivating use.

I don't mind terribly, so you can consider this as a sort of nitpick :blush:

Reply:
This file is shared between Ruby and Python, and this predicate is used on the Ruby side (and will be Python when we share `ApiGraphShared.qll` with Python).

Reply:
Oh, I see :+1: